<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1156</width>
    <height>842</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="optionBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>107</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToMinimumContentsLength</enum>
    </property>
    <item>
     <property name="text">
      <string>original</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>a</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>b</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>c</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>d</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>e</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>f</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>g</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>h</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>i</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>新建项目</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="open_image">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open file</string>
    </property>
   </widget>
   <widget class="QLabel" name="playzone">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>1111</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fileNameLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>file name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>760</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Replay</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>760</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>760</y>
      <width>501</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="threshold_type">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Binary</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Binary Inverted</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Threshold Truncated</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Threshold to Zero</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Threshold to Zero Inverted</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="threshold_type_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Threshold Type:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="threshold_value">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>61</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="thread_value_label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Threshold Value:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1156</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
